src/plugins/data_source/server/plugin.ts

```diff
@@ -32,5 +49,20 @@ export class DataSourcePlugin implements Plugin<DataSourcePluginSetup, DataSourc
     return {};
   }
 
-  public stop() {}
+  public stop() {
+    this.dataSourceService!.stop();
+  }
+
+  private createRouteHandlerContext = (
```

Comment:
move this function to context related file

Reply:
will do